installer/x86_64/install.sh

```diff
@@ -406,6 +419,15 @@ else
             rm -rf $f
         fi
     done
+else
+    demo_mnt="build_raw_image_mnt"
+    demo_dev=$cur_wd/"%%OUTPUT_RAW_IMAGE%%"
```

Comment:
in line 44, you have sourced onie-image.conf, which defines OUTPUT_RAW_IMAGE, here you should use that instead of doing sed